Sdf pointer (for sdfRequired) (#143)
ietf-wg-asdf/SDF: sdf.md

defines for which declarations the corresponding data are mandatory in a
Thing modeled by the current definition.

The value of "sdfRequired" is an array of references, each indicating
one or more declarations that are mandatory to be represented.

References in this array can be SDF names (JSON Pointers), or one of
two abbreviated references:

* a text string with an affordance name.
All affordance declarations that are in the same sdfObject and that
carry this name (there can be multiple ones, one per affordance type)
are declared to be mandatory to be represented.
* the Boolean value `true`.

> The affordance itself that carries the sdfRequired keyword is declared

Reviewer: How about 'sdfRequired true' in Object/Thing definition

Author: Should be fixed now; introducing "groupings" as a way to talk about sdfObject and sdfThing in one word.

defines for which declarations the corresponding data are mandatory in a
Thing modeled by the current definition.

The value of "sdfRequired" is an array of references, each indicating
one or more declarations that are mandatory to be represented.

References in this array can be SDF names (JSON Pointers), or one of

> two abbreviated references:

Reviewer: ```suggestion
two abbreviated reference formats:
```

* a text string with an affordance name or grouping name.

> All affordance declarations that are in the same grouping and that

Reviewer: Suggest to limit this to affordances and groupings that are directly part of the grouping where the sdfRequired is 
```suggestion
All affordance declarations that are directly (i.e., not nested further in another grouping) in the same grouping and that
```

carry this name (there can be multiple ones, one per affordance type)

> are declared to be mandatory to be represented; analogously for

Reviewer: Would find slightly more clear
```suggestion
are declared to be mandatory to be represented. The same applies for
```
